src/app/lightning/lookup-invoice-by-hash.ts

```diff
@@ -1,9 +1,23 @@
 import { LndService } from "@services/lnd"
+import { WalletInvoicesRepository } from "@services/mongoose"
+import { RepositoryError } from "@domain/errors"
 
-export const lookupInvoiceByHash = async (
-  paymentHash: PaymentHash,
-): Promise<LnInvoiceLookup | ApplicationError> => {
+export const lookupInvoiceByHash = async ({
+  hash,
```

Comment:
why did you rename this argument? We've been consistently naming it `paymentHash` everywhere. If you don't rename it you can also delete line 19-20...